proxy.config.json

```diff
@@ -4,7 +4,7 @@
     "secure": true,
     "changeOrigin": true,
     "pathRewrite": {
-      "^/api": "/api/1d59cef4373c4cda8ee3f784bb0c4ef3"
+      "^/api": "/api/382dec296f2f4a3995fa4ef638a8c07c"
```

Comment:
you can try to install `dotenv` and provide in there an API value. See if it makes any sense at all to make it work.

Also, add to README.md information about API. How to start an application with the generated API key.

Reply:
i saved the key value in environment var and used an interceptor to add it to url, kept proxy only to origin change